### PR TITLE
Testing Update 1.1.4

### PR DESCRIPTION
**Pull request by oufmilo**
## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by oufmilo** [comment]
!testme

**Comment by oufmilo** [comment]
!testme

File: manifest.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "requirements": {
         "yunohost": ">= 4.1.3"
     },
-    "multi_instance": true,
+    "multi_instance": false,
```
[the remark this answers is not in the document]
Comment (oufmilo) [reply manifest.json:18]: Just to try if my previous test failed because multi instance is active.
I don't understand why

**Comment by oufmilo** [comment]
!testme

File: manifest.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (yalh76) [reply manifest.json]: test with ci-apps-dev use to fail but multi instance test work on ci-apps

**Pull request merged**